app/src/main/java/chat/rocket/android/fragment/server_config/LoginContract.java

```diff
@@ -23,13 +26,13 @@
 
         void showTwoStepAuth();
 
-        void goBack();
+        void goBack(Context ctx);
     }
 
     interface Presenter extends BaseContract.Presenter<View> {
 
         void login(String username, String password);
 
-        void goBack();
+        void goBack(Context ctx);
```

Comment:
Here's fine to have a Context as parameter since the presenter doesn't have a **direct** access to the view's implementation (would require an ugly casting). :+1: